tests/_files/source_with_multiline_constant_return.php

```diff
@@ -244,4 +244,17 @@ public function Spaceship(): int
             1
         ;
     }
+
+    public function VariableInArray(): int
+    {
+        $n = 1;
+
+        return
+            [
+                2 + $n,
```

Comment:
added extended test with heavy new lines in https://github.com/sebastianbergmann/php-code-coverage/pull/949

`2 + $n` expr itself must not be covered as such expr is unstable and the presence in coverage depends on opcache cache, it makes sense, as `2 + 1` is constant expression that can be evaluated at compile time

Reply:
So you mean is the variable in the array should be covered, as well the variable plus a number shouldn't be covered?
This makes me a little confused.

Reply:
the definition of "executable line" is not simple, but in short, it must address two things:
- when we say about something it is "an executable line", it must be always in xdebug coverage
- it must be stable, see https://github.com/sebastianbergmann/php-code-coverage/issues/889

Reply:
Thanks, I got it